.rubocop_fixme.yml

```diff
@@ -23,7 +23,7 @@ Metrics/AbcSize:
 
 Metrics/ModuleLength:
   Exclude:
-    - 'app/controllers/concerns/hyrax/collections_controller_behavior.rb'
+    - 'app/controllers/concerns/hyrax/dashboard/collections_controller_behavior.rb'
```

Comment:
More comments from slack...

jcoyne: 
> he (@ggeisler) cares about whether the url is `/admin/collections` or `/dashboard/collections`
> And it’s confusing to have modules use different namespaces from the conventions

mjgiarlo:
> we're currently in an inconsistent state, it seems?
> Put otherwise, should @elrayle's PR be consistent with this? https://github.com/samvera/hyrax/blob/master/config/routes.rb#L124  Or this? https://github.com/samvera/hyrax/blob/master/config/routes.rb#L200

elrayle:
> Here is the distinction that I groked from our original conversation.  
> * Managing admin_sets, users, workflows, settings, etc is done by an admin. (use `/admin/`)
> * Managing works, collections, viewing stats can be done by ordinary users. (use `/dashboard`)

mjgiarlo:
> The move from admin to dashboard makes sense to me given that we went from a rigid distinction between "admin dashboard" and "user dashboard" to having a single dashboard that shows users information based on their abilities.
> That sounds reasonable to me, @elrayle.

elrayle:
> Seems to me that works management should have a similar refactor away from hyrax/concerns/generic_work to hyrax/dashboard/works/generic_work.

@ggeisler Want to provide your perspective on  collections under admin vs dashboard?  Potential works refactor is not on the table right now.